### PR TITLE
PAS-614: RMQ, keep alerts with apps

### PR DESCRIPTION
**Pull request by bradenwright**
Keep alerts with config

### CODE REVIEW

File: stable/rabbitmq-ha/templates/alerts.yaml [hunk 1]
```diff
@@ -0,0 +1,71 @@
+apiVersion: v1
```
Comment (bradenwright) [inline stable/rabbitmq-ha/templates/alerts.yaml]: TODO: create a alerts.enable variable

File: stable/rabbitmq-ha/templates/alerts.yaml [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+  rabbitmq.rules: |-
+    groups:
+    - name: rabbitmq-rules
+      rules:
+      - alert: RabbitMqClusterNodeDown
+        expr: rabbitmq_up == 0
+        for: 5m
+        labels:
+          severity: critical
+          team: devops
```
Comment (bradenwright) [inline stable/rabbitmq-ha/templates/alerts.yaml]: remove team devops, either delete or move to variable

File: stable/rabbitmq-ha/templates/alerts.yaml [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Release.Name }}-rabbitmq-alerts
+  namespace: monitoring
+  labels:
+    app: "rabbitmq"
+    chart: {{ .Chart.Name }}-{{ .Chart.Version }}
+    heritage: {{ .Release.Service }}
+    prometheus: kube-prometheus
```
Comment (benjigoldberg) [inline stable/rabbitmq-ha/templates/alerts.yaml]: Make this overridable

File: stable/rabbitmq-ha/templates/alerts.yaml [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+    release: {{ .Release.Name }}
+    role: alert-rules
```
Comment (benjigoldberg) [inline stable/rabbitmq-ha/templates/alerts.yaml]: Make this overridable

**Pull request merged**